src/Main.java

```diff
@@ -1,5 +1,129 @@
+
+import java.util.Arrays;
+import java.util.Scanner;
+
 public class Main {
     public static void main(String[] args) {
-        System.out.println("Hello world!");
+
+        //Zadatak 1.
+
+        Scanner reader = new Scanner(System.in);
+
+        System.out.println("Enter the size of the Array: ");
+
+        int sizeOfArray = reader.nextInt();
+        int[] numbersInArray = new int[sizeOfArray];
+
+        for (int i = 0; i < sizeOfArray; i++) {
+            System.out.println("Enter a number for position " + i + ": ");
+            numbersInArray[i] = reader.nextInt();
+        }
+
+        System.out.println("The numbers in Array are: ");
+        for (int i = sizeOfArray - 1; i >= 0; i--) {
+            if (numbersInArray[i] % 2 ==0) {
+                System.out.println(numbersInArray[i] + " number is even.");
+            }else {
+                System.out.println(numbersInArray[i] + " number is odd.");
+            }
+        }
+
+        //Zadatak 2.
+
+        int[] basicArray = {76543210, 6543210, 543210, 43210, 3210, 210, 10, 1, -1, -10, -210, -3210, -43210, -543210, -6543210};
+        String[] positionArray = {"1st: ", "2nd: ", "3rd: ", "4th: ", "5th: ", "6th: ", "7th: ", "8th: ", "9th: ",
+                                    "10th: ", "11th: ", "12th: ", "13th: ", "14th: ", "15th: "};
+
+        if (basicArray.length <= 15) {
+            for (int i = 0; i < basicArray.length; i++) {
+                System.out.println( positionArray[i] + basicArray[i]);
+            }
+        }else {
+            System.out.println("Basic array has more than 15 positions!");
+        }
+
+        //Zadatak 3.
+
+        Scanner scanner = new Scanner(System.in);
```

Comment:
You can declare the scanner at the beggining once, no need to reinitialize it

Reply:
Ok.